packages/node_modules/@ciscospark/spark-core/src/spark-core.js

```diff
@@ -222,11 +222,14 @@ const SparkCore = AmpState.extend({
         }
         return Promise.resolve();
       })
-      .then(() => Promise.all([
-        this.boundedStorage.clear(),
-        this.unboundedStorage.clear()
-      ]))
-      .then(() => this.credentials.logout(...args));
+      // we do not need to clear the boundedStorage here since we delete @
```

Comment:
Clearing credentials doesn't necessarily clear all of bounded storage. For example, wdm also writes to boundedStorage. Any plugin may write to bounded storage in the future, so we must clear all of it.